macros/NEXT100_muons.config.mac

```diff
@@ -23,23 +22,37 @@
 /Geometry/Next100/pressure 15 bar
 /Geometry/Next100/gas enrichedXe
 /Geometry/Next100/elfield false
-/Geometry/Next100/lab_walls true
```

Comment:
I would keep this line, although commented, since it is necessary when using `HALLA_OUTER` as a vertex region.